pages/index.tsx

```diff
@@ -56,7 +56,15 @@ export default function Index() {
                 easier, please use at your own risk. The source code can be
                 viewed{" "}
                 <a
-                  href="https://github.com/adrianmcli/emp-tools"
+                  href="https://github.com/UMAprotocol/emp-tools"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  here
+                </a>
+                . UMA's main Github can be viewed{" "}
```

Comment:
Can you also add a link to the doc site? Maybe also our discord?

Reply:
I think this should go in the footer perhaps? what do yo think?

Reply:
Agreed on adding to our footer. This can go in a follow up PR